src/theme/components/tab.modifier.ts

```diff
@@ -0,0 +1,18 @@
+import { Components, Theme } from '@mui/material';
+import { GRAY } from '../colors';
+
+export const MuiTab: Components<Theme>['MuiTab'] = {
+  styleOverrides: {
+    root: ({ theme }) => ({
+      backgroundColor: 'transparent',
+      color: theme.palette.text.primary,
+      '&.Mui-selected': {
+        backgroundColor: 'transparent',
+        color: theme.palette.mode === 'dark' ? GRAY : GRAY
```

Comment:
same here?

Reply:
Sure, I've set it to none. I couldn't find a token that makes it transparent so I updated it to 'none'.